swarm/runtime/types/macro_types.py

```
@@ -236,6 +268,16 @@ def default(cls) -> "RunPlanSpec":
             max_total_flows=20,
         )
 
+    def clone(self) -> "RunPlanSpec":
```

Ruff (0.15.11)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)